src/js/themes/base.js

```diff
@@ -378,6 +378,58 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // fontWeight: undefined,
         // extend: undefined,
       },
+      // size: {
+      //   xsmall: {
+      //     textDecoration: undefined,
+      //     fontWeight: undefined,
+      //     extend: undefined,
```

Comment:
Align with base.d.ts

```suggestion
      //     color: undefined,
      //     fontWeight: undefined,
      //     textDecoration: undefined,
```